conda

```diff
@@ -13,10 +13,28 @@ conda create --name 'ENVNAME' python=3.10
 # To install from requirements.txt
 conda install --file requirements.txt
 
+# To clone an environment
+conda create --clone 'ENVNAME' -n 'NEWENV'
+
 # To activate a virtual environment
 conda activate 'ENVNAME'
 
+# To deactivate a virtual environment
+conda deactivate
+
 # To remove and environment
 conda env remove --name 'ENVNAME'
 # or
 conda env remove --prefix /path/to/env
+
+# To list all packages in an environment
+conda list -n 'ENVNAME'
```

Comment:
```suggestion
conda list --name "<environment_name>"
```

- It's better to use long options because they easily memorable for users.
- We can use more descriptive placeholder names here.